options/locale/locale_en-US.ini

```
@@ -1759,7 +1759,7 @@ dashboard.delete_missing_repos_success = All repositories missing their Git file
 dashboard.delete_generated_repository_avatars = Delete generated repository avatars
 dashboard.delete_generated_repository_avatars_success = Generated repository avatars were deleted.
 dashboard.git_gc_repos = Garbage collect all repositories
-dashboard.git_gc_repos_success = All repositories have finished garbage collection.
+dashboard.git_gc_repos_success = Garbage collection started for all repositories.
```

Perhaps its convenient to change the key since the meaning has changed as well?

---

it should have been changed. I am an idiot.

```suggestion
dashboard.git_gc_repos_started = Garbage collection started for all repositories.
```